src/Commands/RunCommand.cs

```diff
@@ -0,0 +1,128 @@
+        public string CashboxConfigurationFile { get; set; } = null!;
+
+
+        public async Task<int> InvokeAsync(InvocationContext context)
+        {
+            var cashboxLauncherConfiguration = JsonSerializer.Deserialize<LauncherConfigurationInCashBoxConfiguration>(await File.ReadAllTextAsync(CashboxConfigurationFile))?.LauncherConfiguration;
+            var launcherConfiguration = GetDefaultLauncherConfiguration();
+
+            MergeLauncherConfiguration(JsonSerializer.Deserialize<LauncherConfiguration>(await File.ReadAllTextAsync(LauncherConfigurationFile)) ?? new LauncherConfiguration(), launcherConfiguration);
+            MergeLauncherConfiguration(cashboxLauncherConfiguration, launcherConfiguration);
+            MergeLauncherConfiguration(ArgsLauncherConfiguration, launcherConfiguration);
+
+            var cashboxConfiguration = JsonSerializer.Deserialize<ftCashBoxConfiguration>(await File.ReadAllTextAsync(CashboxConfigurationFile)) ?? throw new Exception("Empty Configuration File");
+
+            var builder = WebApplication.CreateBuilder();
+            builder.Host
+                .UseSerilog((hostingContext, services, loggerConfiguration) => loggerConfiguration.AddLoggingConfiguration(services, launcherConfiguration.CashboxId.ToString()))
+                .UseConsoleLifetime()
+                .ConfigureServices((hostContext, services) =>
+                {
+                    services.AddSingleton(_ => launcherConfiguration);
+                    services.AddSingleton(_ => cashboxConfiguration);
+                    services.AddSingleton(_ => new Dictionary<Guid, ProcessHostMonarch>());
+                    services.AddHostedService<ProcessHostMonarcStartup>();
+                });
+
+            if (launcherConfiguration.LauncherPort == null)
+            {
+                throw new Exception("Launcher port cannot be null.");
+            }
+            builder.WebHost.ConfigureKestrel(options => HostingService.ConfigureKestrel(options, new Uri($"http://[::1]:{launcherConfiguration.LauncherPort!}")));
+
+
+            builder.Services.AddCodeFirstGrpc();
+
+            var app = builder.Build();
+
+            app.UseRouting();
+            app.UseEndpoints(endpoints => endpoints.MapGrpcService<ProcessHostService>());
+
+            await app.StartAsync();
+
+
+            await app.WaitForShutdownAsync();
+
+            return 0;
+        }
+
+        private static LauncherConfiguration GetDefaultLauncherConfiguration()
+        {
+            return new LauncherConfiguration
+            {
+                LauncherPort = 3000,
+                Sandbox = false,
+                UseOffline = false,
+                LogFolder = Path.Join(Paths.ServiceFolder, "logs"),
+                LogLevel = LogLevel.Information,
+                ServiceFolder = Paths.ServiceFolder,
+                PackagesUrl = new Uri("https://packages.fiskaltrust.cloud"),
+                DownloadTimeout = 15,
```

Comment:
I'd name this property `DownloadTimeoutSec`, and in general follow that pattern when we use `int` for timespans :D